main/main.cpp

```diff
@@ -31,8 +31,12 @@
 #include <windows.h>
 #endif
 
+#include <iostream>
+
 int main(int argc, char** argv)
 {
+    std::cout << "Hello world, I am MuseScore!!\n";
```

Comment:
Serious? This is what `qDebug()` is for

Reply:
Was a problem, Windows build doesn't start, without any messages. qDebug() doesn't print messages to console. 
I will check this again and will fix it if not so.